presto-main/src/test/java/com/facebook/presto/operator/scalar/TestRegexpFunctions.java

```diff
@@ -141,6 +141,75 @@ public void testRegexpReplace()
         assertFunction("REGEXP_REPLACE(VARCHAR 'x', '.*', 'xxxxx')", createUnboundedVarcharType(), "xxxxxxxxxx");
     }
 
+    @Test
+    public void testRegexpReplaceLambda()
+    {
+        // One or more matches with non-empty, not null capturing groups
+
```

Comment:
Drop the blank line; same for all other places.

Reply:
David asked me to add the empty lines....

Reply:
I didn't find his comment. Where is it exactly?

Reply:
Oh I misunderstood his comment. I thought he asked to add an empty line after the comments.